src/ui/components/Icon/styles.scss

```diff
@@ -115,3 +115,8 @@
 .Icon-featured {
   @include icon($name: "featured");
 }
+
+.Icon-restart,
```

Comment:
> It is not used, but I added this alias for consistency purpose.

How do you mean? I don't see any other styles like this in `Icon/styles.scss`. If it's unused I don't see why we need this style.